tpm/tpm_test.go

```diff
@@ -498,3 +498,15 @@ func TestTakeOwnership(t *testing.T) {
 		t.Fatal("Couldn't take ownership of the TPM:", err)
 	}
 }
+
+func TestForceClear(t *testing.T) {
+	// Only enable this if you know what you're doing.
+	t.Skip()
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+
+	// This test code assumes that the owner auth is the well-known value.
```

Comment:
I don't think this comment is accurate: TPM_ForceClear always succeeds if TPM_DisableForceClear has not been called, and it doesn't perform any owner auth checks, according to Section 6.3 of [the TPM 1.2 spec, part 3](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Main-Part-3-Commands_v1.2_rev116_01032011.pdf).

So, maybe something like "Note that ForceClear does not check owner auth." Or just drop the comment here and make sure the description in the Skip is good.

Reply:
Done, leftover